packages/workbox-broadcast-cache-update/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "workbox-broadcast-cache-update",
   "version": "0.0.2",
-  "description": "A service worker helper library that uses the Broadcast Channel API to announce when two Response objects differ.",
+  "description": "A service worker helper library that uses the BroadcastChannel API to announce when two Response objects differ.",
```

Comment:
The usage I typically go with:
Broadcast Channel API &mdash;if referring generally to the API. (So the old version is correct.)
`BroadcastChannel` object